content/guides/erp-connection/step1-who-is-your-connector-for.md

```diff
@@ -0,0 +1,101 @@
+:::
+
+:::info
+In the PIM, product information that comes from the ERP are usually stored in dedicated **read only** attributes and in a specific **attribute group**.
+This allows Julia to have the information at her fingertips without the risk of mixing it with her own PIM product information.
+:::
+
+## Personas' needs
+
+Below, you will find the main needs, associated with each user profile, in terms of processes, documentations, actions related to your future ERP connector.
+
+### Julia's needs
+
+**If the product is initially created in the PIM**
+
+Julia needs an easy way to create the product in her PIM, and exchanges product information with the ERP solution.
+
+So she needs:
+
+* An **easy process** to create dynamically a SKU for the product
+
+* An **easy way** to tag her new product so that your ERP connector can automatically export the corresponding product data from her PIM to the ERP solution
+
+* A **friendly other process** to retrieve ERP product information she needs in return
+
+* A **strategy** to define the periodicity of these automatic import/export processes
+
+* Perhaps also a **friendly Connector UI** to trigger manually all processes (import/export) if needed (full or partial imports)
+
+* A **friendly UI** to log all processes and a notification system to be alerted when something goes wrong.
+
+**If the product is initially created in the ERP, she needs:**
+
+Julia needs an easy way import product information from the ERP solution to her PIM.
+
+So she needs:
+
+* An **easy to follow process** to import automatically ERP product data into her PIM.
+
+* An **easy to follow strategy** to define the periodicity of this automatic import process.
+
+* Perhaps also a **friendly Connector UI** to trigger manually the import process if needed (full or partial imports)
+
+* A **friendly UI** to log all automatic and manual import processes and a notification system to be alerted when something goes wrong.
+
+### Peter and/or Filips' needs
+
+Peter and/or Filips needs a easy way to install, configure and maintain your connector.
+
+So they need:
+
+* An **easy to follow download/installation process:** to retrieve and install your connector.
+It depends on your business model. If you require your client to install the connector themself, make sure you provide appropriate documentation.
+
+* A **full documentation** about **the limits of the connector** in terms of **scalability** and **performance** (your connector limits).
+
+* An **easy to follow configuration process** to set up the connection between the ERP Solution and Akeneo PIM as well as parameters to configure the connector. It requires documentation to explain the methodology on how to configure your connector compared to your business needs.
+
+* Sometime, a **friendly import process** to perform the first import of product data: due to the volume of data usually imported, this may require special support from your end.
```

Comment:
```suggestion
* Sometimes, a **friendly import process** to perform the first import of product data: due to the volume of data usually imported, this may require special support from your end.
```